src/test/java/nextstep/subway/acceptance/StationAcceptanceTest.java

```diff
@@ -11,75 +15,122 @@
-    @DisplayName("지하철역을 조회한다.")
-    @Test
-    void getStations() {
-    }
-
-    /**
-     * Given 지하철역을 생성하고
-     * When 그 지하철역을 삭제하면
-     * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
-     */
-    // TODO: 지하철역 제거 인수 테스트 메서드 생성
-    @DisplayName("지하철역을 제거한다.")
-    @Test
-    void deleteStation() {
-    }
+class StationAcceptanceTest {
+
+	private static final int CREATED = HttpStatus.CREATED.value();
+
+	@LocalServerPort
+	int port;
+
+	@BeforeEach
+	public void setUp() {
+		RestAssured.port = port;
+	}
+
+	/**
+	 * When 지하철역을 생성하면
+	 * Then 지하철역이 생성된다
+	 * Then 지하철역 목록 조회 시 생성한 역을 찾을 수 있다
+	 */
+	@DisplayName("지하철역을 생성한다.")
+	@Test
+	void createStation() {
+		// When 지하철역을 생성하면
+		String stationName = "강남역";
+		ExtractableResponse<Response> response = createStation(stationName);
+
+		// Then 지하철역이 생성된다
+		assertThat(response.statusCode()).isEqualTo(CREATED);
+
+		// Then 지하철역 목록 조회 시 생성한 역을 찾을 수 있다
+		List<String> allStationNames = getAllStationNames();
+		assertThat(allStationNames).containsAnyOf(stationName);
+	}
+
+	/**
+	 * Given 2개의 지하철역을 생성하고
+	 * When 지하철역 목록을 조회하면
+	 * Then 2개의 지하철역을 응답 받는다
+	 */
+	@DisplayName("지하철역을 조회한다.")
+	@Test
+	void getStations() {
+
+		// Given 2개의 지하철역을 생성하고
+		String stationFirst = "강남역";
+		String stationSecond = "역삼역";
+		createStation(stationFirst);
+		createStation(stationSecond);
+
+		// When 지하철역 목록을 조회하면
+		List<String> allStationNames = getAllStationNames();
+
+		// Then 2개의 지하철역을 응답 받는다
+		assertAll(
+			() -> assertThat(allStationNames).hasSize(2),
+			() -> assertThat(allStationNames).containsAnyOf(stationFirst),
+			() -> assertThat(allStationNames).containsAnyOf(stationSecond)
+		);
+
+	}
+
+	/**
+	 * Given 지하철역을 생성하고
+	 * When 그 지하철역을 삭제하면
+	 * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
+	 */
+	@DisplayName("지하철역을 제거한다.")
+	@Test
+	void deleteStation() {
+		// Given 지하철역을 생성하고
+		String stationName = "강남역";
+		ExtractableResponse<Response> createResponse = createStation(stationName);
+
+		// When 그 지하철역을 삭제하면
+		deleteStation(createResponse.jsonPath().getString("id"));
+
+		// Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
+		List<String> allStationNames = getAllStationNames();
+		assertThat(allStationNames).isEmpty();
+	}
+
+	private ExtractableResponse<Response> createStation(String stationName) {
+		Map<String, String> params = new HashMap<>();
+		params.put("name", stationName);
+
+		return RestAssured.given().log().all()
+			.body(params)
+			.contentType(MediaType.APPLICATION_JSON_VALUE)
+			.when().post("/stations")
+			.then().log().all()
+			.extract();
+	}
+
+	private ExtractableResponse<Response> getAllStations() {
+		return RestAssured.given().log().all()
+			.when().get("/stations")
+			.then().log().all()
+			.extract();
+	}
+
+	private List<String> getAllStationNames() {
+		return getAllStations()
+			.jsonPath()
+			.getList("name", String.class);
+	}
+
+	private void deleteStation(String id) {
+		RestAssured.given().log().all()
+			.when()
+			.delete("stations/" + id)
```

Comment:
```suggestion
			.when().delete("/stations/" + id)
```
사실 사소하지만 얘만 다른 url 인 것 같아서요ㅎㅎ

Reply:
# 😲😲